src/OpenLoco/Objects/LandObject.h

```diff
@@ -35,6 +35,8 @@ namespace OpenLoco
         uint8_t var_0E;
         uint8_t pad_0F[0x16 - 0x0F];
         uint32_t var_16;
+        uint8_t var_1B;
+        uint8_t var_1C;
```

Comment:
should we have a `static_assert` for this struct (`LandObject`) size? seems like 2 bytes were just added here, that sounds strange to me with my limited knowledge of these structs

Reply:
The objects are a bit weird in that i don't know their sizes for sure until we implement the object load functions. And sometimes even later on due to how the loads work. (This is in contrast to almost every other struct).

Reply:
ah ok, so the struct sizes aren't fully known yet, that makes sense then. so how do you know when an struct *is* fully known?

Reply:
The load function usually reveals all.

Reply:
I went and read the load function and noticed a mistake.